dags/openshift_nightlies/scripts/utils/connect_to_platform.sh

```diff
@@ -47,7 +47,10 @@ setup(){
     export KUBECONFIG=/home/airflow/workspace/config
     curl -sS https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar xz oc
     export PATH=$PATH:$(pwd)
-
+    if [[ $(oc get nodes | grep -c worker) -lt 1 ]]; then 
+        echo "No worker nodes to install grafana agent"
```

Comment:
is this a scenario which you have come across?

Reply:
https://github.com/cloud-bulldozer/airflow-kubernetes/issues/144 so we might be migrating the grafana agent to infra nodes , will that have an impact for hypershift?